tests/jsTests/menuSpec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Suite : Tests for showing and hiding of menu
+ * Description : This is a test suite for showing and hiding menu for smaller screens.
```

Comment:
Already done, please check it here https://github.com/systers/FirstAide-web/pull/262/commits/aac881a07ce238abf294b5b87077278a091b9db9